webpack.config.js

```diff
@@ -61,6 +61,8 @@ module.exports = options => {
 
   const entry = {
     main: path.join(__dirname, "/src/index.js"),
+    interactionTests: path.join(__dirname, "/src/tests/interactions-utils"),
+    testIds: path.join(__dirname, "/src/tests/base-tests-utils"),
```

Comment:
testIds is not very accurate name e.g. if we want to add to base-tests-utils something else in the future...
let's maybe call it baseTestUtils?
And are we sure we need two separate entries at all here?